### PR TITLE
Check LS launcher for ability to launch

### LINKED ISSUE #5528: [REGRESSION] Error message is shown when try to open file
If try to open file in the stack where this file type language server is disabled, the error message will be shown.

**Reproduction Steps:**
- Create Java project
- Create create and run json (ts, python, php) file 
- Pay attention to the screen 

**Expected result:**
File will be opened

**Observed result:**
Error message is shown and file has not opened

**OS and version:**    
Che - 5.15.0 ; Chrome - 59.0 ; Fedora - 25;

**Attachment:**
eclipse che workspace3xndsy

### PR DESCRIPTION
**Pull request by tsmaeder**
This PR fixes https://github.com/eclipse/che/issues/5528 by filtering contributed languages and launchers for their ability to launch.

### CODE REVIEW

**Comment by benoitf** [comment]
ci-build

**Comment by benoitf** [comment]
@tsmaeder Could you please add a changelog entry
https://github.com/eclipse/che/wiki/Development-Workflow#pull-requests

> We always require that each PR has:
A descriptive title.
A link to the issue that initiated the PR if one exists.
Changelog: one line summary in markdown for the release's Changelog.

The template is showing that per default

File: wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImpl.java [hunk 1]
```diff
@@ -10,6 +10,17 @@
  *******************************************************************************/
 package org.eclipse.che.api.languageserver.registry;
 
+import static org.eclipse.che.api.languageserver.shared.ProjectLangugageKey.createProjectKey;
```
Comment (benoitf) [inline wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImpl.java]: The order or imports is not expected
https://github.com/codenvy-legacy/dev/wiki/Source-file#imports

Comment (tsmaeder) [reply wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImpl.java]: Confusingly, the doc mentions the package groups in one order, then defines their order as the exact reverse of the one a couple of lines above.

**Comment by vparfonov** [comment]
thanks @tsmaeder  I've merged it here https://github.com/eclipse/che/pull/5533

**Pull request closed without merging**